packages/devtools/lib/src/ui/flutter_html_shim.dart

```diff
@@ -14,7 +14,14 @@ String fontStyleToCss(TextStyle textStyle) {
   return sb.toString();
 }
 
+final Map<Color, String> cssColors = {};
```

Comment:
make this private as it is an implementation detail.

Reply:
done